src/Measure.php

```diff
@@ -184,6 +186,8 @@ class Measure
         ],
         self::VOLUME => [
             CubicMetre::class,
+            CubicCentimeter::class,
+            CubicMillimeter::class,
```

Comment:
I think it might be best to keep spelling conventions consistent. 🙂 

1. update the **class names** to use international spelling, instead of American – _e.g.,_ `CubicCentimetre`
2. update the **file names** to match